modules/reindex/src/test/java/org/elasticsearch/reindex/ReindexIdTests.java

```diff
@@ -53,6 +70,16 @@ private ClusterState stateWithIndex(Settings.Builder settings) {
         return ClusterState.builder(ClusterState.EMPTY_STATE).metadata(Metadata.builder(Metadata.EMPTY_METADATA).put(meta)).build();
     }
 
+    private Settings.Builder standardSettings() {
+        return Settings.builder().put(IndexSettings.MODE.getKey(), IndexMode.STANDARD);
```

Comment:
maybe randomly return an empty setting? Since standard is the default.

Reply:
:+1: